spec/features/npq_separation/admin/applications_spec.rb

```diff
@@ -188,4 +188,33 @@
     expect(page).to have_css("h1", text: "Application for #{outcome.user.full_name}")
     expect(page).to have_css(".govuk-notification-banner--success", text: /rescheduled/i)
   end
+
+  scenario "changing lead provider approval status" do
+    application = create(:application, :accepted)
+
+    visit npq_separation_admin_application_path(application)
+
+    expect(page).to have_css("h1", text: "Application for #{application.user.full_name}")
+
+    within(".govuk-summary-list:first-of-type") do |summary_list|
+      expect(summary_list).to have_summary_item("Lead provider approval status", "Accepted")
+      expect(summary_list).to have_link("Change to pending")
+
+      click_link("Change to pending")
+    end
+
+    expect(page).to have_css("h1", text: "Are you sure you want to change the status to pending?")
+    choose "No", visible: :all
+    click_button "Change status to Pending"
+
+    expect(page).to have_css(".govuk-error-message", text: "Confirm you wish to change the status back to pending")
```

Comment:
I'm maybe missing something here, but this seems slightly off?

- Create accepted application
- Click change to pending
- Select **no** (so application is still accepted?)
- Click change to pending again
- Message prompts to confirm we want to change **back** to pending (but we didn't change it in the first instance?)